CMakeLists.txt

```diff
@@ -88,10 +88,25 @@ macro(jerry_add_compile_flags)
   jerry_add_flags(CMAKE_C_FLAGS ${ARGV})
 endmacro()
 
+# SHOULD THIS BE, SAY, AT THE TOP OF THE FILE?
+if(CMAKE_C_COMPILER_ID MATCHES "GNU")
+  set(USING_GCC 1)
+  set(USING_GCC_OR_CLANG 1)
+endif()
+
+if(CMAKE_C_COMPILER_ID MATCHES "Clang")
+  set(USING_CLANG 1)
+  set(USING_GCC_OR_CLANG 1)
```

Comment:
I think the USING_GCC_OR_CLANG variable is unnecessary, instead you should check if the compiler is a variant of gcc or a Clang compiler.